indigo/distribution.yaml

```diff
@@ -11242,6 +11242,17 @@ repositories:
       url: https://github.com/GT-RAIL/robot_pose_publisher.git
       version: develop
     status: maintained
+  robot_process:
+    doc:
+      type: git
+      url: https://github.com/snt-robotics/robot_process.git
+      version: master
+    source:
+      test_pull_requests: true
```

Comment:
Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, please either remove this line or provide the necessary access as described in the wiki page.

Reply:
Sorry, I forgot about this step. I generated a webhook as I can't give admin access to the org

Reply:
:+1: